Formula/x11vnc.rb

```diff
@@ -40,7 +40,7 @@ def install
   end
 
   test do
-    system "#{bin}/x11vnc --version"
+    system "#{bin}/x11vnc", "--version"
```

Comment:
Now that the arguments are separated: We prefer `bin/"x11vnc"` over `"#{bin}/x11vnc"`.